<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FakeVim::Internal::FakeVimOptionPage</class>
 <widget class="QWidget" name="FakeVim::Internal::FakeVimOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>474</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="checkBoxUseFakeVim">
     <property name="text">
      <string>Use FakeVim</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="behaviorBox">
     <property name="title">
      <string>Vim Behavior</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="6" column="0">
         <widget class="QCheckBox" name="checkBoxStartOfLine">
          <property name="text">
           <string>Start of line</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkBoxHlSearch">
          <property name="text">
           <string>Highlight search results</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBoxSmartIndent">
          <property name="text">
           <string>Smart indentation</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBoxSmartTab">
          <property name="text">
           <string>Smart tabulators</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBoxExpandTab">
          <property name="text">
           <string>Expand tabulators</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBoxIncSearch">
          <property name="text">
           <string>Incremental search</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBoxUseCoreSearch">
          <property name="text">
           <string>Use search dialog</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QCheckBox" name="checkBoxSmartCase">
          <property name="text">
           <string>Use smartcase</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="checkBoxWrapScan">
          <property name="text">
           <string>Use wrapscan</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="checkBoxPassControlKey">
          <property name="toolTip">
           <string>Passes key sequences like Ctrl-S to Qt Creator core instead of interpreting them in FakeVim. This gives easier access to Qt Creator core functionality at the price of losing some features of FakeVim.</string>
          </property>
          <property name="text">
           <string>Pass control key</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBoxAutoIndent">
          <property name="text">
           <string>Automatic indentation</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBoxIgnoreCase">
          <property name="text">
           <string>Use ignorecase</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="checkBoxShowCmd">
          <property name="text">
           <string>Show partial command</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="checkBoxPassKeys">
          <property name="toolTip">
           <string>Lets Qt Creator handle some key presses in insert mode so that code can be properly completed and expanded.</string>
          </property>
          <property name="text">
           <string>Pass keys in insert mode</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="checkBoxShowMarks">
          <property name="text">
           <string>Show position of text marks</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QCheckBox" name="checkBoxRelativeNumber">
          <property name="toolTip">
           <string>Displays line numbers relative to the line containing text cursor.</string>
          </property>
          <property name="text">
           <string>Show line numbers relative to cursor</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="labelShiftWidth">
          <property name="text">
           <string>Shift width:</string>
          </property>
          <property name="buddy">
           <cstring>spinBoxShiftWidth</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxShiftWidth">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>80</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelTabulator">
          <property name="toolTip">
           <string>Vim tabstop option.</string>
          </property>
          <property name="text">
           <string>Tabulator size:</string>
          </property>
          <property name="buddy">
           <cstring>spinBoxTabStop</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxTabStop">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>80</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelScrollOff">
          <property name="text">
           <string>Scroll offset:</string>
          </property>
          <property name="buddy">
           <cstring>spinBoxScrollOff</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxScrollOff"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelBackspace">
          <property name="text">
           <string>Backspace:</string>
          </property>
          <property name="buddy">
           <cstring>lineEditBackspace</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditBackspace"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelIsKeyword">
          <property name="text">
           <string>Keyword characters:</string>
          </property>
          <property name="buddy">
           <cstring>lineEditIsKeyword</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditIsKeyword"/>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBoxReadVimRc">
          <property name="text">
           <string>Read .vimrc from location:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="Utils::PathChooser" name="pathChooserVimRcPath" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonCopyTextEditorSettings">
          <property name="text">
           <string>Copy Text Editor Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSetQtStyle">
          <property name="text">
           <string>Set Qt Style</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSetPlainStyle">
          <property name="text">
           <string>Set Plain Style</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkBoxUseFakeVim</tabstop>
  <tabstop>checkBoxAutoIndent</tabstop>
  <tabstop>checkBoxSmartIndent</tabstop>
  <tabstop>checkBoxExpandTab</tabstop>
  <tabstop>checkBoxSmartTab</tabstop>
  <tabstop>checkBoxHlSearch</tabstop>
  <tabstop>checkBoxShowCmd</tabstop>
  <tabstop>checkBoxStartOfLine</tabstop>
  <tabstop>checkBoxPassKeys</tabstop>
  <tabstop>checkBoxIncSearch</tabstop>
  <tabstop>checkBoxUseCoreSearch</tabstop>
  <tabstop>checkBoxIgnoreCase</tabstop>
  <tabstop>checkBoxSmartCase</tabstop>
  <tabstop>checkBoxWrapScan</tabstop>
  <tabstop>checkBoxShowMarks</tabstop>
  <tabstop>checkBoxPassControlKey</tabstop>
  <tabstop>checkBoxRelativeNumber</tabstop>
  <tabstop>spinBoxShiftWidth</tabstop>
  <tabstop>spinBoxTabStop</tabstop>
  <tabstop>spinBoxScrollOff</tabstop>
  <tabstop>lineEditBackspace</tabstop>
  <tabstop>lineEditIsKeyword</tabstop>
  <tabstop>checkBoxReadVimRc</tabstop>
  <tabstop>pathChooserVimRcPath</tabstop>
  <tabstop>pushButtonCopyTextEditorSettings</tabstop>
  <tabstop>pushButtonSetQtStyle</tabstop>
  <tabstop>pushButtonSetPlainStyle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
